app/src/androidTest/java/ch/epfl/sweng/eventmanager/test/repository/MockEventsRepository.java

```diff
@@ -89,7 +89,7 @@
                 "} ]\n";
 
 
-        TypeToken<List<Zone>> zonesToken = new TypeToken<List<Zone>>() {
+        TypeToken<List<Zone>> zoneToken = new TypeToken<List<Zone>>() {
```

Comment:
Can be deleted as `List<Zone>` becomes `Zone` so `TypeToken` (for composite types) can be replaced by `Zone `direcltly.